src/Uno.UI/UI/Xaml/Controls/Popup/Popup.cs

```diff
@@ -33,12 +33,6 @@ internal FlyoutBase AssociatedFlyout
 		}
 	}
 
-	/// <summary>
-	/// In WinUI, Popup has IsTabStop set to true by default.
-	/// UWP does not include IsTabStop, but Popup is still focusable.
-	/// </summary>
-	private protected override bool IsTabStopDefaultValue => true;
```

Comment:
After a lot of testing, Popup is in fact not IsTabStop and therefore, not focusable. I've testing on WinUI with ComboBox, Flyout, MenuFlyout, ContentDialog and CalendarDatePicker. All their popups are not IsTabStop and explicitly trying to `TryFocusAsync` will fail. Instead, it's the job of the first focusable element inside the popup (which is focused when the popup is opened) to deal with e.g. Tab and Escape. This PR is therefore blocked by #13634 which fixes focusing inside the popup which is necessary (I suppose among other things) for CalendarDatePicker's popup to close on Escape.

I'm also guessing that we can remove a good amount of code from Popup which deals with focus and KeyDown since the Popup will no longer be focusable.